### DISCUSSION TITLE
Broken URL in documentation

### DISCUSSION THREAD

**Discussion by lasharela** [hyperledger/fabric#2431] [General]
```
[`getCreator()` chaincode API](https://github.com/hyperledger/fabric-chaincode-node/blob/{BRANCH}/fabric-shim/lib/stub.js#L293).
If Go is used as a chaincode language, the [client identity chaincode library](https://github.com/hyperledger/fabric-chaincode-go/blob/{BRANCH}/pkg/cid/README.md)
```
This URL is giving a 404 error page.

https://github.com/hyperledger/fabric/edit/release-2.2/docs/source/developapps/analysis.md

I don't know the correct URL unfortunately.

**Comment by lindluni** [comment]
@lasharela

Thank you for bringing this to our attention, can you open a Jira item for this at https://jira.hyperledger.org
